### PR TITLE
Add CI tests using Github actions

### PR DESCRIPTION
**Pull request by skateinmars**
As the title says.

This will help when accepting external contributions :)

### CODE REVIEW

File: sqalx_test.go [hunk 1]
```diff
@@ -35,6 +37,7 @@ func TestSqalxConnectPostgreSQL(t *testing.T) {
 func TestSqalxConnectMySQL(t *testing.T) {
 	dataSource := os.Getenv("MYSQL_DATASOURCE")
 	if dataSource == "" {
+		fmt.Println("DEBUGDEBUG - skipping due to empty MYSQL_DATASOURCE")
```
Comment (sixstone-qq) [inline sqalx_test.go:40]:
```suggestion
		t.Log("DEBUGDEBUG - skipping due to empty MYSQL_DATASOURCE")
```

File: sqalx_test.go [hunk 2]
```diff
@@ -24,6 +25,7 @@ func prepareDB(t *testing.T, driverName string) (*sqlx.DB, sqlmock.Sqlmock, func
 func TestSqalxConnectPostgreSQL(t *testing.T) {
 	dataSource := os.Getenv("POSTGRESQL_DATASOURCE")
 	if dataSource == "" {
+		fmt.Println("DEBUGDEBUG - skipping due to empty POSTGRESQL_DATASOURCE")
```
Comment (sixstone-qq) [inline sqalx_test.go:28]:
```suggestion
		t.Log("DEBUGDEBUG - skipping due to empty POSTGRESQL_DATASOURCE")
```

**Review by sixstone-qq — COMMENTED** [review]
Great job! :clap: Just one detail, could you use `t.Log` instead to work with redirections?

File: sqalx_test.go (hunk 1 above)
Comment (skateinmars) [reply sqalx_test.go]: I didn't want to commit this :see_no_evil: 
Cleaned up the text and switched to `t.log` :ok_hand:

**Pull request merged**